tests/phpunit/tests/media.php

```diff
@@ -3828,9 +3961,9 @@ public function data_wp_get_loading_attr_default_before_and_no_loop() {
 	 *
 	 * @ticket 56930
 	 * @ticket 58548
+	 * @ticket 58235
 	 *
 	 * @covers ::wp_filter_content_tags
```

Comment:
When removing a reference to the deprecated function being covered, it should be replaced with a `covers` annotation for the new function.
```suggestion
	 * @covers ::wp_filter_content_tags
	 * @covers ::wp_img_tag_add_loading_optimization_attrs
	 * @covers ::wp_get_loading_optimization_attributes
```